packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.js

```diff
@@ -58,18 +60,19 @@ const defaultProps = {
   helperText: null,
   usernameLabel: 'Username',
   usernameValue: '',
-  onChangeUsername: () => undefined,
+  onChangeUsername: null,
   isValidUsername: true,
   passwordLabel: 'Password',
   passwordValue: '',
-  onChangePassword: () => undefined,
+  onChangePassword: null,
```

Comment:
Can we keep this as undefined?

Reply:
Sure we can.

The reason I changed it from `() => undefined` to `null` is that I can't compare functions but `null` and `undefined` I can.